src/containers/shared/components/Transaction/AMM/AMMBid/parser.tsx

```diff
@@ -0,0 +1,17 @@
+import {
+  getAMMAccountID,
+  getAuthAccounts,
+  getMinBid,
+} from '../../../../MetaParser'
+
+export function parser(tx: any, meta: any) {
+  const accountID = getAMMAccountID(meta)
```

Comment:
Wasn't amm id removed?

Reply:
it has been removed from the top - but I'm still including it as a row in the simple details since it still exists in the metadata. We can remove this if necessary (should ask jasmine)

Reply:
Nvm this is the account not the amm_id which was another identifier.